protx-client/src/components/ProTx/components/charts/ChartInstructions.css

```diff
@@ -35,6 +44,11 @@
   font-size: var(--font-size-normal);
   font-weight: var(--font-weight-light);
 }
+.report-instructions-steps-group-condensed * {
```

Comment:
is the * needed?

Reply:
I can see this is following the already existing `report-instructions-steps-group`. so ignore my comment.